postgis/lwgeom_functions_basic.c

```diff
@@ -2806,3 +2806,26 @@ Datum ST_Scale(PG_FUNCTION_ARGS)
 
   PG_RETURN_POINTER(ret);
 }
+
+Datum ST_Points(PG_FUNCTION_ARGS);
+PG_FUNCTION_INFO_V1(ST_Points);
+Datum ST_Points(PG_FUNCTION_ARGS)
+{
+	if (PG_ARGISNULL(0))
+	{
+		PG_RETURN_NULL();
```

Comment:
just a note: you don't need to do that when the function is defined as STRICT, it's automatic.